channels/unsorted.m3u

```diff
@@ -313,8 +313,6 @@ https://news18lokmat-lh.akamaihd.net/i/n18lokmat_1@178974/index_5_av-p.m3u8
 http://92.60.237.32:1935/live/nisville/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Noovo
 https://bcsecurelivehls-i.akamaihd.net/hls/live/551061/618566855001/master.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",OAN Encore
```

Comment:
The unsorted.m3u still contains the tvg-name="" tag?

Reply:
Yes, we will need to update the file later.

Reply:
Oh, okay.